drivers/include/tmp006.h

```diff
@@ -101,6 +101,20 @@ extern "C"
 #endif
 
 /**
+ * @brief   Default low power mode
+ */
+#ifndef TMP006_USE_LOW_POWER 
+#define TMP006_USE_LOW_POWER (0)
+#endif
+
+/**
+ * @brief   Default raw value mode
```

Comment:
Raw value mode

If 0, measurements will be converted to Celsius. If 1, raw adc readings will be returned.